lib/plaid.js

```diff
@@ -3,7 +3,19 @@
 var environments = require('./plaidEnvironments.js');
 var Client = require('./PlaidClient.js');
 
+var isPlaidError = function(err) {
+  return (
+    err != null &&
+    typeof err.error_type === 'string' &&
+    typeof err.error_code === 'string' &&
+    typeof err.error_message === 'string' &&
+    (err.display_message == null ||
```

Comment:
this should be `err.display_message === null || typeof err.display_message === 'string'`, otherwise it could return true when `display_message` is missing, which I don't think we want here. The schema for a plaid error will always have the field present.

Reply:
got it — i previously assumed undefined would also be a valid option.